exporter/datadogexporter/README.md

```diff
@@ -50,6 +50,12 @@ The exact values for `send_batch_size` and `send_batch_max_size` depends on your
 - Log intake: https://docs.datadoghq.com/api/latest/logs/
 - Metrics V2 intake: https://docs.datadoghq.com/api/latest/metrics/#submit-metrics
 
+### Fall back to the Zorkian metric client with feature gate
```

Comment:
IMO we should mention that this is temporary and that users will have to manually upgrade on a future minor version when we graduate this flag

Reply:
SG, added a note below.